Integrations/integration-Pipl.yml

```diff
@@ -0,0 +1,280 @@
+                    //HumanReadable: tableToMarkdown('Pipl', data, columns),
+                    //HumanReadable: data,
+                    EntryContext: ec
+                };
+    };
+
+    var buildEC = function(data, i) {
+        return {
+                    Addresses : data[i].Addresses,
+                    Email : data[i].Email,
+                    IDs : data[i].UserIDs,
+                    Names: data[i].Names,
+                    Phones: data[i].Phones,
+                    Usernames: data[i].Usernames
+                };
+    };
+
+    var addPerson = function(person) {
+        var response = {
+            Names: '',
+            Phones: '',
+            Gender: '',
+            DoB: '',
+            Image: '',
+            Usernames: '',
+            Email: '',
+            Educations: '',
+            UserIDs: '',
+            URLs: '',
+            Jobs: '',
+            Addresses: ''
+        };
+        if (person.names) {
+            for (var i=0; i<person.names.length; i++) {
+                response['Names'] += person.names[i].middle ? person.names[i].first + ' ' + person.names[i].middle + ' ' + person.names[i].last + '\n' : person.names[i].first + ' ' + person.names[i].last + '\n';
+            }
+        }
+        if (person.phones) {
+            for (var i=0; i<person.phones.length; i++) {
+                response['Phones'] += person.phones[i].display + ' ' + person.phones[i].display_international + '\n';
+            }
+        }
+        if (person.gender){
+            response['Gender'] = person.gender.content;
+        }
+        if (person.dob) {
+            response['DoB'] = person.dob.display;
+        }
+        if (person.images) {
+            //response['Image'] = '![some alt]('+person.images[0].url+'=size=100X100)';
+            var url='https://thumb.pipl.com/image?height=100&width=100&favicon=false&zoom_face=false&token='+person.images[0].thumbnail_token;
+            response['Image']='__img__:src,'+ url +';alt,'+''+';height,100;width,100';
+
+        }
+        if (person.usernames) {
+            for (var i=0; i<person.usernames.length; i++) {
+                response['Usernames'] += person.usernames[i].content + '\n';
+            }
+        }
+        if (person.emails) {
+            response['Email'] = [];
+            for (var i=0; i<person.emails.length; i++) {
+                response['Email'][i] = {Address:person.emails[i].address};
+            }
+        }
+        if (person.educations) {
+            for (var i=0; i<person.educations.length; i++) {
+                response['Educations'] += person.educations[i].display + '\n';
+            }
+        }
+        if (person.user_ids) {
+            for (var i=0; i<person.user_ids.length; i++) {
+                response['UserIDs'] += person.user_ids[i].content + '\n';
+            }
+        }
+        if (person.urls) {
+            for (var i=0; i<person.urls.length; i++) {
```

Comment:
format properly your JS code.` for (var i=0; i<person.urls.length; i++) {` => 
`for (var i = 0; i < person.urls.length; i++) {`